src/components/oscal-utils/OSCALLinkUtils.js

```diff
@@ -32,3 +32,16 @@ export default function resolveLinkHref(
     )
   );
 }
+
+export function getURLMediaType(url) {
+  let fileExtension = "";
+  try {
+    fileExtension = new URL(url).pathname.split(".").pop();
+    if (fileExtension === "/") {
+      fileExtension = "href";
+    }
```

Comment:
An href (like "https://github.com/") returns the media type "/". I think it would make more sense for this to be "HREF", so I added a conditional check here.

Reply:
I would strongly prefer that this ends up as “Unknown”

Reply:
Also, `"Unknown"` is becoming a [magic string](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants). We should create a variable for this.

```
const fallbackFileExtension = "Unknown";
```

which we should use just about everywhere within this function.